validator/validator.go

```diff
@@ -44,11 +44,13 @@ func New(ctx context.Context, logger *zap.Logger, validatorShare *collections.Va
 	msgQueue := msgqueue.New()
 	ibfts := make(map[beacon.Role]ibft.IBFT)
 	ibfts[beacon.RoleAttester] = ibft.New(
+		logger,
 		ibftStorage,
 		network,
 		msgQueue,
 		&proto.InstanceParams{
 			ConsensusParams: proto.DefaultConsensusParams(),
+			IbftCommittee:   validatorShare.Committee,
```

Comment:
we sure there is no place that using that?

Reply:
I've added an issue to refactor ibft committee #67